### ISSUE TITLE
Popup window not closing

### ISSUE THREAD

**Issue by pors** [xinranxiao/meteor-accounts-spotify#5]
In a browser all works fine, I can login into spotify, and the popup window closes when completed. In a mobile app (iPhone with cordova) the login also works fine, but I have to close the popup manually by clicking "Done". The window is completely blank.

Any idea how to debug this?

**Comment by pors** [comment]
BTW it does work in the emulator and also on a locally connected device (via USB). The problem exists only with a deployed version via TestFlight

**Comment by xinranxiao** [comment]
Hey sorry for the late response -- not completely sure how to debug this. This seems like a very niche case -- where is the server running via your deployed version? Is the redirect URL, etc. set up properly?

**Comment by pors** [comment]
Yeah everything works fine (redirect URL etc). I'm using the exact same setup for web, simulator, device with USB cable, and testflight version. All work except the testflight version. But I don't have a developer console for that...

**Comment by Benzopower** [comment]
Hello i need your help directly, i dod know how or how i did remove or hide my 2 Spotify artist profiles i had one with over 700k streams and the other one is 800k. That have take my long time bro please can you hjälp me to get them back or maybe find out a some solution. 

Can you write an email to me or how an we talk with each other???
